i3pystatus/wireguard.py

```diff
@@ -0,0 +1,81 @@
+from i3pystatus import IntervalModule
+from i3pystatus.core.command import run_through_shell
+
+__author__ = 'Pluggi'
+
+
+class OpenVPN(IntervalModule):
```

Comment:
Probably want to change this class name.

Reply:
Done! :D